vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher.cc

```diff
@@ -333,7 +330,7 @@ void Publisher::SaveVisitInternal(
         this,
         _1);
 
-    ledger_->database()->SavePublisherInfo(std::move(publisher_info), callback);
+    ledger_->database()->SavePublisherInfo(publisher_info->Clone(), callback);
```

Comment:
Curious: why was this changed to clone? `publisher_info` does not appear to be used after this point.

Reply:
Good catch, there was another use of it in there but it was removed recently. Fixed.